src/components/molecules/Definition/TestkubeTheme.ts

```diff
@@ -0,0 +1,187 @@
+import Colors from '@styles/Colors';
```

Comment:
Shouldn't we abstract this file as a SyntaxHighlightTheme to a higher level? (I am sure we'll face more place where we'll need Syntax Highlighting in the future)

Reply:
Of course, this is WIP PR. Still a lot of things to complete. For convenience left it there)